### DISCUSSION TITLE
Directus in issue

### DISCUSSION THREAD

**Discussion by zuber56** [directus/directus#6078] [Archive]
I have table create and field a select in text area then P tag automatically with field data of creating in the database include the P tag. 
let you can see. you are changing directus website.
so I hope will be changing your code.

**Comment by benhaynes** [comment]
I'm a little lost here. Could you try to explain this again?

**Reply by zuber56** [reply]
let you can see.
https://prnt.sc/144ql6r

**Reply by rijkvanzanten** [reply]
Don't use the WYSIWYG (HTML) interface if you don't want it to save HTML 👍🏻

**Reply by zuber56** [reply]
https://prnt.sc/14gysmt

**Reply by zuber56** [reply]
https://prnt.sc/14gzopt

**Reply by zuber56** [reply]
https://prnt.sc/14h62e6

**Reply by zuber56** [reply]
In this regarding video give me tutorials.
plz 
thank you for my reply.

**Reply by zuber56** [reply]
Hello Developers,

Would you please provide me flow or information regarding how to manage the relationship between the 2 tables. I am facing some issues when I provide a relationship between two tables. The official documents have also not mention well. Please help me as soon as possible.
